conda/environments/builddocs.yml

```diff
@@ -4,8 +4,8 @@ channels:
 - nvidia
 - conda-forge
 dependencies:
-# required for building docs
```

Comment:
This comment didn't seem to be adding any information (this is already a file called `builddocs.yml`), so proposing removing it. I'll undo this if reviewers disagree.